html/admin/index.html

```diff
@@ -147,6 +154,11 @@
         <h1>System</h1>
       </div>
       <p class="lead">System Functions</p>
+        <form class="simple_form form-inline" id="form_logrotate">
```

Comment:
The logs are rotated automatically, so I don’t think there’s a need to expose the functionality to the administrator, which would allow us to remove this and the API endpoint.